packages/webamp/js/components/MainWindow/index.tsx

```diff
@@ -106,10 +106,11 @@ const MainWindow = React.memo(({ analyser, filePickers }: Props) => {
           />
           <Time />
         </div>
-        <Visualizer
+        {/* <Visualizer
           // @ts-ignore Visualizer is not typed yet
           analyser={analyser}
-        />
+        /> */}
```

Comment:
Fine to just delete this code if it's not used any more

Reply:
Was just waiting for your word :saluting_face: